scripts/getGitCommit.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+if command -v git &>/dev/null && git rev-parse &>/dev/null; then
+	GIT_COMMIT=$(git rev-parse --short HEAD)
+	if [ -n "$(git status --porcelain --untracked-files=no)" ]; then
+		GIT_COMMIT="$GIT_COMMIT-unsupported"
+	fi
+    echo $GIT_COMMIT
```

Comment:
Inconsistent indentation on lines 8 and 9 vs. 4-7

Check out [shfmt](https://github.com/mvdan/sh) if you want an easy way to automatically tidy up scripts.  We use this on auth_proxy and install

Reply:
wth is are tabs doing in there :/